src/main/java/org/broadinstitute/hellbender/tools/exome/TargetTableColumns.java

```diff
@@ -13,10 +13,10 @@
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
 public enum TargetTableColumns {
-    NAME("NAME", true),
-    CONTIG("CONTIG", true),
-    START("START", true),
-    END("END", true),
+    NAME("name", true),
```

Comment:
I could tell you why this will work, or I could run the darn workflow and tell you what happens.  Hmmm, I wonder what the responsible thing to do would be?